test/unit/ui_utils_spec.js

```diff
@@ -627,4 +628,17 @@ describe("ui_utils", function () {
       });
     });
   });
+
+  describe("calcRound", function () {
+    it("", function () {
```

Comment:
This test needs a name :-)
```suggestion
    it("should handle different browsers/environments correctly", function () {
```